src/python/bot/minimizer/grammars/JavaScriptBaseLexer.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Translation from Java code for JavaScriptBaseLexer made to work with
+JavaScriptLexer"""
+
```

Comment:
Is this also an auto-generated file, or did you accidentally miss my comments below?

Reply:
Maybe you forgot to upload the changes you made locally :)

Reply:
Sorry. Messed the comments from before. This is not an auto-generated file. The JavaScriptLexer is an auto-generated file, but it was generated to work with this JavaScriptBaseLexer file which was not available in python. I had to translate it from Java. I can change the import, but the fact that the naming conventions unusual I can't change without changing the auto generated file. Even though this file isn't necessary auto-generated, I'm going to leave it with the rest of the code not being linted.